contents/localization.html

```diff
@@ -0,0 +1,4 @@
+<h1>Localization</h1>
+<p>Firefox products are developed for a global audience. More than 70 percent of Firefox users live outside of North America. Our products are localized for more than 80 languages and regional dialects. 
+<p>Meeting the needs of our users around the world and working to inject more Firefox personality into copy will always be a balancing act. Copy with idioms, humor or complex sentence structures may pose challenges for localizers; your pithy English phrase may be misunderstood, or require 50 percent more copy to translate effectively. Worse, it may simply not be appropriate for a region’s sensibilities. The good news is that writing well for the web &mdash; short sentences, clear hierarchy, non-ambiguous language &mdash; will go far to support our localization efforts. <a ref="https://wiki.mozilla.org/L10n:WorldReady">Visit the localization team’s wiki</a> for more advice on the specifics of writing for localization. 
```

Comment:
Oh, and the link on this line starts `<a ref="`, but should start `<a href="`.  🙂